metrics.go

```diff
@@ -0,0 +1,217 @@
+package cke
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+
+	v3 "github.com/coreos/etcd/clientv3"
+	"github.com/cybozu-go/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+type logger struct{}
+
+func (l logger) Println(v ...interface{}) {
+	log.Error(fmt.Sprint(v...), nil)
+}
+
+const (
+	namespace     = "cke"
```

Comment:
The name `namespace` is too generic for the package `cke`.
Consider creating `metrics` package and move this file under it.
Splitting this file should also be considered as this file is a bit large.